assets/template-collection.css

```diff
@@ -227,6 +227,10 @@
   color: var(--color-foreground-75);
 }
 
+.disclosure-has-popup[open] > .facets__summary::before {
+  z-index: 3;
+}
```

Comment:
I wonder if we should handle the `click outside` to close the menus with JS. And keep this CSS approach for a no JS experience. 
The drawer menu doesn't seem to have a `click outside closes the menu` kinda thing and when using the `facets__summary` it could reduce the amount of clicks to open others. 
But might be overkill 🤷  Just think it would like a smoother experience.

Reply:
I think this might be too much complexity on the JS side, personally.  There's also more risk of a "time-warp" sort of effect if we do it this way.  Remember that when filters are changed with JS enabled, the other details elements get re-rendered.  If we were to re-render an element after they've clicked the element to open it, then it will close when it re-renders.

We might be able to get around that concern by re-rendering the contents of the details element instead of the whole element, which might be one option. 🤔 

We can add it to the issue as an idea and think about it.